config/routes.rb

```diff
@@ -6,14 +6,7 @@
   end
 
   if Settings.features.streaming_media
-    # download file_name must NOT include format extension (e.g.  .mp4)
-    # so insist on . being URL encoded (see media routing specs)
-    constraints file_name: %r{[^/\.]+} do
-      get '/media/:id/:file_name' => 'media#download', as: :media
-      get '/media/auth/:id/:file_name' => 'webauth#login_media_download', as: :auth_media_download
-    end
-
-    # stream file_name must include format extension, eg .../oo000oo0000.mp4/stream
+    # stream file_name must include format extension, eg .../oo000oo0000.mp4/verify_token
     #  other dots do not need to be URL encoded (see media routing specs)
```

Comment:
Ah, I was thinking that there were additional tests in that file for the other routes.  I'll add routes back in that check one of the other paths so that can keep those checks in somewhere.